### ISSUE TITLE
Task 3.3.1: Add functionality for remaining Zcash RPCs to Indexer trait and implement for StateService and FetchService.

### ISSUE THREAD

**Issue by idky137** [zingolabs/zingo-proxy#137]
The Following Zcash RPCs will need to be implemented in Zaino:
- [getaddressdeltas](https://zcash.github.io/rpc/getaddressdeltas.html)
- [getblockcount](https://zcash.github.io/rpc/getblockcount.html)
- [getblockdeltas](https://zcash.github.io/rpc/getblockdeltas.html)
- [getblockhash](https://zcash.github.io/rpc/getblockhash.html)
- [getblockhashes](https://zcash.github.io/rpc/getblockhashes.html)
- [getblockheader](https://zcash.github.io/rpc/getblockheader.html)
- [getchaintips](https://zcash.github.io/rpc/getchaintips.html)
- [getdifficulty](https://zcash.github.io/rpc/getdifficulty.html)
- [getmempoolinfo](https://zcash.github.io/rpc/getmempoolinfo.html)
- [getspentinfo](https://zcash.github.io/rpc/getspentinfo.html)
- [gettxout](https://zcash.github.io/rpc/gettxout.html)
- [gettxoutsetinfo](https://zcash.github.io/rpc/gettxoutsetinfo.html)

Wrapper functionality for the following Zcash RPCs in Zebra will also need to be implemented in Zaino:
- [getmininginfo](https://zcash.github.io/rpc/getmininginfo.html)
- [getnetworksolps](https://zcash.github.io/rpc/getnetworksolps.html)
- [getnetworkinfo](https://zcash.github.io/rpc/getnetworkinfo.html)
- [getpeerinfo](https://zcash.github.io/rpc/getpeerinfo.html)
- [ping](https://zcash.github.io/rpc/ping.html)

- Blocked on #54

**Comment by zancas** [comment]
Perhaps we should consider prioritizing Block-Explorer functionality, and moving other RPCs to a subsequent grant?

**Comment by idky137** [comment]
This list is incomplete. The full list of RPC services to be added to Zaino is detailed in https://github.com/zingolabs/zaino/issues/196. 

I am closing this issue as it is a duplicate of that one.
